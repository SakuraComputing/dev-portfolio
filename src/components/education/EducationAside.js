import React from 'react';

export const EducationAside = () => (
    <aside className="education aside section">
        <div className="section-inner">
            <h2 className="heading">Education</h2>
            <div className="content">
                <div className="item">
                    <h3 className="title"><i className="fa fa-graduation-cap"></i> BSc Computer Science</h3>
                    <h4 className="university">University of Bristol <span className="year">(2008-2011)</span></h4>
                </div>
            </div>
        </div>
    </aside>
);

export default EducationAside;