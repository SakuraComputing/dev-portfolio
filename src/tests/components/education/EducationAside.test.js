import React from 'react';
import { shallow } from 'enzyme';
import EducationAside from '../../../components/education/EducationAside';

describe('EducationAside Section', () => {
    it('should render correctly', () => {
        const education = jest.fn();
        const wrapper = shallow(<EducationAside education={education}/>);
        expect(wrapper).toMatchSnapshot();
    });
});