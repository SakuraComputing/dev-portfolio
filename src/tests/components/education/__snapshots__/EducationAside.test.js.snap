// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EducationAside Section should render correctly 1`] = `
<aside
  className="education aside section"
>
  <div
    className="section-inner"
  >
    <h2
      className="heading"
    >
      Education
    </h2>
    <div
      className="content"
    >
      <Item
        item={
          Object {
            "id": 1,
            "institution": "Leeds University",
            "qualifiation": "BSc Degree",
            "year": "(1989-1991)",
          }
        }
        key="1"
      />
      <Item
        item={
          Object {
            "id": 2,
            "institution": "South Trafford College",
            "qualifiation": "A Levels (BBC)",
            "year": "(1988-1989)",
          }
        }
        key="2"
      />
    </div>
  </div>
</aside>
`;
